Various fixes and style improvements

exander77 opened the pull request:
I updated my style to current version.

cdot commented:
I've ported all these up to 3.1.0. Happy with all except bf562c45fd26bc3de4669767605d8ad9176e16d3 - I'd like to understand how a game with undefined `whosTurnKey`  can be in any state other than WAITING.

exander77 commented:
@cdot I have no idea, I don't really have any patches on my end other than using my CSS style and I have this on every refresh of games page. I removed all old games, just created new game, added bot and myself and voila I have an error. The error does not prevent me to do anything, but it really bugs me. I debugged it a little and it is really undefined. It then leads to accessing email on undefined.
